app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    ls = command.split()
```

Comment:
The variable name 'ls' is not descriptive. According to the checklist, you should use descriptive and correct variable names. Consider renaming it to something like 'command_parts'.